command.json

```diff
@@ -35,6 +35,13 @@
           "description": "Company address",
           "required": false,
           "prompt": true
+        },
+        {
+          "name": "adminPassword",
```

Comment:
still thinking this better as kebab case: `admin-password` ... no?

Reply:
Can you also use `admin-password` as a flag/parameter you pass along on the CLI? I assumed here (when you create a new bedrock project), that `adminPassword` is only used and referred to in JS (hence camelCase).

Reply:
I changed it to kebab case.